modules/ting/ting.admin.inc

```diff
@@ -136,7 +136,7 @@ function ting_admin_reservable_settings($form_state) {
   $form['reservation_settings'] = array(
     '#type' => 'fieldset',
     '#title' => t('Reservation settings'),
-    '#tree' => FALSE,
```

Comment:
If I remember correctly I found that it was unnecessary, since [#tree is FALSE by default](https://www.drupal.org/docs/7/api/form-api/tree-and-parents)

But maybe just keep it for clarity?